### PR TITLE
Add NFT Article in Participate

### PR DESCRIPTION
**Pull request by huhn511**
This article explains the different types of NFTs on IOTA and how you can participate.

Please feel free to provide input or feedback! 

--- Spacial thanks to @just4fun94

## Type of change

Choose a type of change, and delete any options that are not relevant.
- Documentation Enhancement

### CODE REVIEW

File: internal/participate/use-cases/nfts.md [hunk 1]
```diff
@@ -1,27 +1,41 @@
...
+There are many different use cases for NFTs, the most common use case is to link an image to it which becomes unqiue digital art which can be sold on open NFT marketplaces.
 
-## Heading 2 (Sub topic 1)
+Other use cases would be to use NFts as assets in games, for event tickets, for Decentraliced Finance (DeFi), for votings in dezentralced autonomous organisations (DAOs) or for any objects in the Metaverse.
 
-content
+## Different NFT Types on IOTA 
 
-## Heading 2 (Sub topic 2)
+- Layer 1 Assets also known as "Digital Native Assets". With the [IOTA tokenization framework](https://blog.iota.org/tokenization-on-the-tangle-iota-digital-assets-framework/) you can create NFTs on L1 using the NFT output type. Layer 1 NFTs can be transfered on the whole layer without any fees and enable you to transfer NFTs between different smart contract chains.
```
Comment (Critical94) [inline internal/participate/use-cases/nfts.md:15]: Maybe additional h3 headlines for the different NFT types like `### Digital Native Assets` and `### Layer 2 Assets` are more clear than this `ul`

File: internal/participate/use-cases/nfts.md [hunk 2]
```diff
@@ -1,27 +1,41 @@
...
+The IOTA NFT marketplace is a feeless alternative to the expensive and energy-wasting approaches taken by other networks.
+
+Their tests phase has ended. For the next steps they implement Smart Contracts and integrate Browser Wallets to make the NFT trading secure and accessible.
+
+- https://nftiota.org/
+
+
+## NFT Projects
+
+The community also creates really amazing artworks and already dropping them on the Smart Contrat Testnet. Please note - all these NFTs are on the Testnet, and be careful that some NFT projects might do new drops when the mainnet launches.
```
Comment (Critical94) [inline internal/participate/use-cases/nfts.md:33]:
```suggestion
The community also creates really amazing artworks and already dropping them on the Smart Contract Testnet. Please note - all these NFTs are on the Testnet, and be careful that some NFT projects might do new drops when the mainnet launches.
```

File: internal/participate/use-cases/nfts.md [hunk 3]
```diff
@@ -1,27 +1,41 @@
...
 
-<br/>
+Here is a list of communiy projects. Please feel free to add yours!
 
-**need to add a footer section for further content - (three cards)**
+Project collection page:
+- https://iota-nft.art/
```
Comment (Critical94) [inline internal/participate/use-cases/nfts.md:38]:
```suggestion
- - [iota-nft.art](https://iota-nft.art/)
```

File: internal/participate/use-cases/nfts.md [hunk 4]
```diff
@@ -1,27 +1,41 @@
...
 
-<br/>
+The first IOTA NFT marketplace, was developed by the IOTA community and launched in July 2021 on the IOTA 2.0 DevNet. 
 
----
+The IOTA NFT marketplace is a feeless alternative to the expensive and energy-wasting approaches taken by other networks.
+
+Their tests phase has ended. For the next steps they implement Smart Contracts and integrate Browser Wallets to make the NFT trading secure and accessible.
+
+- https://nftiota.org/
```
Comment (Critical94) [inline internal/participate/use-cases/nfts.md:28]:
```suggestion
- [nftiota.org](https://nftiota.org/)
```

File: internal/participate/use-cases/nfts.md (hunk 1 above)
Comment (Dr-Electron) [reply internal/participate/use-cases/nfts.md:15]: Yeah i think that would be nicer too

**Comment by huhn511** [comment]
Thanks @Critical94 👍

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**